### DISCUSSION TITLE
Text Selection Handles do not appear on Minimal Markdown and Dynamic Markdown (Android)

### DISCUSSION THREAD

**Discussion by catwithbanana** [standardnotes/forum#2808] [General]
When using the Minimal Markdown and Hybrid Markdown editors text selection handles do not show up to help select text. Additionally, scrolling through text by sliding on the spacebar with gBoard doesn't work with these editors. This makes editing nearly impossible on mobile. Plain Text and Basic Markdown work as expected.

I am running Android 12 on a Pixel 5 with gBoard as my keyboard. These are the same editors that have keyboard appearance issues as I previously noted in [this separate issue under the archived mobile repository.](https://github.com/standardnotes/mobile/issues/539)

This issue (no selection handles + keyboard not appearing without a long press) persists with every keyboard I've tried, though some are able to scroll with the space bar as expected.

Please let me know if you need any more info. I have not yet tried uninstalling and reinstalling Standard Notes. Thanks.

[This is reposted from the archived mobile repository -- see the original post for screenshots](https://github.com/standardnotes/mobile/issues/620). Both issues are still occuring on the latest version.

**Comment by catwithbanana** [comment]
These issues persist with the latest update, 3.23.8 (3002010). Minimal Markdown and Hybrid Markdown are not usable on mobile.

**Comment by catwithbanana** [comment]
This issue persists with the latest update, 3.31.20 (3002129). Minimal Markdown and "Markdown" (Markdown Pro) still do not work and are not usable on mobile. "Markdown Alternative" works as expected.

**Comment by dashrandom** [comment]
Again, I confirm I face the same issue with a Sony Xperia 5 III, stock rom.

**Comment by chaserene** [comment]
I'm experiencing the same on LineageOS 19.1 (~Android 12).

if I'm lucky, I can select a single word by double-tapping on it, but no way to extend/shrink the selection. my keyboard (AnySoftKeyboard) has a very good  built-in text selection tool, but even that doesn't work here.

this makes SN semi-dysfunctional for me on mobile. I tried older releases and found that it was version [3.11.0](https://github.com/standardnotes/mobile/releases/tag/3.11.0) that introduced this bug. my current solution is rolling the app back to the last version where this still worked, which is [3.9.12](https://github.com/standardnotes/mobile/releases/tag/3.9.12).

that means this quite visible bug is 7 months old and hasn't been reported for the first 3 months, so I wonder if it's some special factor that triggers it only for a few users (so far  I know about 3), or people just didn't bother to report it.

**Comment by catwithbanana** [comment]
This issue persists with the latest update, 3.32.5 (3002144)

**Comment by dashrandom** [comment]
@chaserene @catwithbanana I've discovered that long tapping in the text selection area of markdown now brings up the typing indicator and the keyboard. Could you test and confirm? Just tapping however still doesn't bring up the keyboard, but it's a massive improvement from not being able to type at all.

**Comment by chaserene** [comment]
@dashrandom as far as I remember, I had no issues bringing up the keyboard. I can't test it for you because I'm back on 3.9.12 (where I have none of the issues mentioned in this thread) and I won't update until these are known to be solved.

(heads up -- as a result of the issue being moved to a new repo, I got a notification about the OP.)

**Comment by chaserene** [comment]
this bug has been present for 1.5 years and has been severely limiting the user experience (stay on a very old version, or lose the ability to select text). can we get an update on this?

also, meta: why was this issue (#1795) turned into a discussion? @amanharwara said [here](https://discord.com/channels/894600255058898974/894600255058898977/1139259778707763200) that bug reports should go into issues.
